File: task_2/hist.py
import pandas as pd
import matplotlib.pyplot as plt

df = pd.read_csv("Housing.csv")

# Histogramme de la colonne 'bedrooms'
plt.figure(figsize=(10, 4))
plt.hist(df['bedrooms'], bins=15, color='skyblue', edgecolor='black')
plt.title('Distribution of bedrooms')
plt.xlabel('beedroom')
plt.ylabel('Fréquence')
plt.grid(True)
plt.show()

File: task_2/hist.R
# Chargement des packages
library(readr)
library(ggplot2)

# Chargement du dataset Housing.csv
data <- read_csv("Housing.csv")
View(data)


# l'histogramme pour 'bedrooms'
histogram <- ggplot(data, aes(x = bedrooms)) +
  geom_histogram(binwidth = 1, fill = "blue", color = "black") +
  labs(title = "Distribution du Nombre de Chambres",
       x = "Nombre de Chambres",
       y = "Fréquence")

print(histogram)